src/analysis/FVA/fluxVariability.m

```diff
@@ -281,19 +295,28 @@
     preCompMinSols((model.lb(Order(Presence)) == relSol)) = {sol};
 end
 %Restrict the reactions to test only those which are not at their boundariestestFv.
-rxnListMin = rxnNameList(~minSolved);
-rxnListMax = rxnNameList(~maxSolved);
+%isNegUnb/isPosUnb were row vectors to match origPresence
+rxnListMin = rxnNameList(~minSolved & ~isNegUnb);
+rxnListMax = rxnNameList(~maxSolved & ~isPosUnb);
 
+Vmax=zeros(nRxns,length(rxnNameList));
+Vmin=zeros(nRxns,length(rxnNameList));
 
 if ~PCT_status && (~exist('parpool') || poolsize == 0)  %aka nothing is active
     if minNorm
         for i = 1:length(rxnNameList)
-        
             LPproblem.osense = 1;
-            [minFlux(i),Vmin(:,i)] = calcSolForEntry(model,rxnNameList,i,LPproblem,0, method, allowLoops,printLevel,minNorm,cpxControl,preCompMinSols{i});
+            if(~isNegUnb(i))

```

Comment:
The calculation should be done in all instances (an unbounded reaction does not necessarily have unbounded) fluxes.  (Imagine A -> B -> C ->D , if A -> B has a maximal bound of 5 all others have a max bound of 5 as well, even if their ub is infinite.)